src/Raven.Client/Util/TcpUtils.cs

```diff
@@ -149,27 +149,27 @@ internal static async Task<(TcpClient Client, string Url)> ConnectAsyncWithPrior
         {
             string selectedUrl = null;
             TcpClient tcpClient = null;
-            
-            foreach (var url in info.TcpServerUrls)
+
+            if (info.TcpServerUrls != null)
```

Comment:
Shouldn't this be `ServerUrls`

Reply:
not sure what's going on here it seems not to fully push my changes...